src/vector/v.maxent.swd/v.maxent.swd.html

```diff
@@ -1,54 +1,80 @@
 <h2>DESCRIPTION</h2>
 
-<em>v.maxent.swd</em> Produce a set of SWD files as input to MaxEnt 3.3.3k
-(version 3.4.1 untested, but should work as well), based on a point vector
-layer(s) of species distribution(s), a point layer with background points and
-user-defined raster layers. The user can also let the function create the
-background point file. (respecting region and mask, if set).
-
-<p>The SWD file format is a simple CSV-like file file format as
-described in Elith et al. 2011. The first three fields provide the
-species name, x-coordinate and y-coordinate, while subsequent fields
-contain the values of the user-selected environmental parameters.
-Note that the coordinates are not used by the MaxEnt program if all
-environmental data is given in the SWD format file.
-
-<p>The user can provide one or more species point files, which will be
-combined in one SWD file. The map names of both the species point
-layers and the environmental parameters can be replaced by alias
-names, which will be used by MaxEnt.
+<em>v.maxent.swd</em> Produce a set of SWD files as input to MaxEnt 
+3.3.3k+, based on a point vector layer(s) of species distribution(s), a 
+point layer with background points and user-defined raster layers. The 
+user can also let the function create the background point file. 
+(respecting region and mask, if set).
+
+<p>
+The SWD file format is a simple CSV-like file file format as described 
+in Elith et al. 2011. The first three fields provide the species name, 
+x-coordinate and y-coordinate, while subsequent fields contain the 
+values of the user-selected environmental parameters. Note that the 
+coordinates are not used by the MaxEnt program if all environmental 
+data is given in the SWD format file.
+
+<p>
+The user can provide one or more species point files, which will be 
+combined in one SWD file. The map names of both the species point 
+layers and the environmental parameters can be replaced by alias names, 
+which will be used by MaxEnt.
+
+<p>
+If alias names are used , a CSV file (alias_file) can be created with 
+alias names in the first column and map names in the second column, 
+separated by comma, without header. This can be used as input in 
+r.maxent.lambdas (but note that this addon does not seem to work with 
+outputs from Maxent 3.4+).
 
 <h2>NOTES</h2>
 
 The SWD output files are simple comma delimited text
 files (.csv) and can therefore be easily read in in for example R
 and subsequently used in other models / functions.
 
+<p>
+This addon was written as a Python alternative to the r.out.maxent_swd 
+script and provides much of the same functionality. The original script 
+has since been ported to Python (see also). One difference is that with 
```

Comment:
Not sure if the history is so relevant for the manual. And there are good reasons for having this addon anyway. So I would drop the intro:
```suggestion
This addon is a vector based alternative to <em>r.out.maxent_swd</em>. It can
be more efficient with sparse data points. The main difference is that with
```